jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/oracle/OracleSqlGenerationVisitor.java

```diff
@@ -21,6 +21,7 @@
     // If set to true, the selectlist elements will get aliases such as c1, c2, ...
     // Can be refactored if we find a better way to implement it
     private boolean requiresSelectListAliasesForLimit = false;
+    private static final String TIMESTAMP_FORMAT = "'HH24:MI:SS.FF3 DD-MM-YYYY'";
```

Comment:
English and US notation for date is usually `YYYY-MM-DD`.

Reply:
Changed the format to: YYYY-MM-DD HH:MI:SS.FF6